src/pages/filter/common/attributes/attribute.store.ts

```diff
@@ -0,0 +1,184 @@
+import { makeAutoObservable, toJS } from 'mobx'
+
+import { ActionType } from '@declarations'
+import { FilterKindEnum } from '@core/enum/filter-kind.enum'
+import { ModeTypes } from '@core/enum/mode-types-enum'
+import dtreeStore from '@store/dtree'
+import filterStore from '@store/filter'
+import { GlbPagesNames } from '@glb/glb-names'
+import activeStepStore from '@pages/filter/dtree/components/active-step.store'
+import modalsControlStore from '@pages/filter/dtree/components/modals/modals-control-store'
+import modalsVisibilityStore from '@pages/filter/dtree/components/modals/modals-visibility-store'
+import {
+  AttributeKinds,
+  TCondition,
+  TNumericConditionBounds,
+  TPropertyStatus,
+  TVariant,
+} from '@service-providers/common/common.interface'
+import { addAttributeToStep } from '@utils/addAttributeToStep'
+import { changeEnumAttribute } from '@utils/changeAttribute/changeEnumAttribute'
+import { getConditionJoinMode } from '@utils/getConditionJoinMode'
+import { getCurrentModeType } from '@utils/getCurrentModeType'
+
+export class AttributeStore {
+  constructor() {
+    makeAutoObservable(this)
+  }
+
+  public get isRefiner(): boolean {
+    return filterStore.method === GlbPagesNames.Refiner
```

Comment:
Can we avoid to use `filterStore.method` for page detecting?
I think, we can some base store and extend it for filter refiner and dtree with specific methods.
And then provide required values to `EnumAttribute` as props (see `components/numeric-condition`)